frontend/src/utils/download/GCalendar.js

```diff
@@ -16,27 +16,36 @@ const GTIME_FORMAT = "YYYY-MM-DDTHH:mm:ssZ";
 const NUM_WEEKS_AHEAD = 5;
 const RECURRING_EVENT_RULE = "RRULE:FREQ=WEEKLY;COUNT=" + NUM_WEEKS_AHEAD;
 
-export function addEvents(subsections) {
+export function addEvents(schedule) {
```

Comment:
Isn't the format for adding to google calendar and pushing to ClassEvents a little different?

Reply:
The whole looping through the classes is the same though, if they differ in some parts but are the same in others, pull out the parts where they are the same

Reply:
Actually upon looking at it more, I do realize that although the looping constructs look similar, you do have to do different things in there. So that's my bad, I should have paid more attention. What I would recommend doing here to avoid repetition is something with lambdas, but I'm not sure if that would be too much work. We can discuss this